README.md

```diff
@@ -9,7 +9,6 @@ The Matching Engine Unity C# SDK provides everything required to create applicat
 ## Prerequisites  
 
 * Unity 2019.2 or newer, along with selected platforms (iOS, Android) for your project
-* .Net Standard 2.0
```

Comment:
i guess this is pending whether we can get 4.x and Mono working right?

Reply:
IL2CPP is the only choice compatible with IOS, so for multiple device target development (most), we may just need a carve out for Android Mono + 4.x likely isn't supported.